src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -69,7 +69,9 @@ var ANONYMOUS = '<<anonymous>>';
 var ReactPropTypes = {
   array: createPrimitiveTypeChecker('array'),
   bool: createPrimitiveTypeChecker('boolean'),
```

Comment:
Leave `PropTypes.bool` and `PropTypes.func` for backwards compatibility. Not sure what route should be taken to deprecate these.